ipv6-custom/Makefile

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2010-2012 OpenWrt.org
+# Modifications from original Copyright (C) 2022 Minim, Inc.
+#
+# This Makefile and the code shipped in src/ is free software, licensed
+# under the GNU Lesser General Public License, version 2.1 and later.
+# See src/COPYING for more information.
+#
+# Refer to src/COPYRIGHT for copyright statements on the source files.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minim-ipv6-custom
+PKG_RELEASE:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
Nit.  I don't think the host-build.mk include or HostBuild call are needed for this package.  Looks like we have some other packages that may needlessly include those too though :)

Reply:
Fixed, am still learning about packages.